test/end-to-end/arns.test.ts

```diff
@@ -24,6 +24,7 @@ import {
 } from 'testcontainers';
 import axios from 'axios';
 import { rimraf } from 'rimraf';
+import { isTestFiltered } from '../utils.js';
```

Comment:
Why import it here if it's not being used?

Reply:
Good catch. I think I added it initially before realizing we didn't need to skip the test in this file.